tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -166,6 +169,19 @@ Verify RHODS Release Version Number
     ${version} =  Get RHODS Version
     Should Match Regexp    ${version}    ^[0-9]+\.[0-9]+\.[0-9]+\(-[0-9]+)*$
 
+Verify That Alert Notification Emails Can Be Updated And Changed In Alertmanager Configuration
```

Comment:
I think the title in polarion is more user friendly: `Verify Users Can Update Notification Email After Installing RHODS With The AddOn Flow`

Reply:
@rkamble5 are you going to implement this changes? The conversation is marked as resolved by there are no comments. re-opening the conversation.

Reply:
Done